### ISSUE TITLE
Do you provide the code to fine-tune the student model?

### ISSUE THREAD

**Issue by MrDadiao** [bboylyg/NAD#3]
Hello,
     I tried to apply your work to my own badnets, and I wonder how do you fine-tune the student model to the teacher model?

**Comment by bboylyg** [comment]
If my understanding is right, your question is "how to finetuning the teacher model ?"   
Specifically, we obtained the teacher model by finetuning all layers of the backdoored model using 5% clean data with data augmentation techniques. The finetuning code is easy to get by just setting all the param Beta = 0, which means the distillation loss to be zero in the training process.  More details please see the `config.py` and setting the param as follows:  
    ```parser.add_argument('--beta1', type=int, default=0, help='beta of low layer')
    parser.add_argument('--beta2', type=int, default=0, help='beta of middle layer')
    parser.add_argument('--beta3', type=int, default=0, help='beta of high layer')```  
After that finetuning the model by the command: ```bash $ python train_badnet.py```.

**Comment by MrDadiao** [comment]
I know what you mean, but I don't see any use of  beta in the train_badnet.py code, I think the command after setting the param might be `bash $ python main.py` or change the line 124 in code train_badnet.py from `train_loader = get_backdoor_loader(opt)`  to `train_loader = get_train_loader(opt)`? 

Actually, I tried these steps, but I got great results which is close to NAD after finetuning in this way, so I wanna ask you for the finetuning code you used.

**Comment by bboylyg** [comment]
Note that `train_badnet.py` is only used to train a backdoored model as the baseline. The code of `main.py` is used both to finetuning the teacher model and to implement attention distillation.

**Comment by MrDadiao** [comment]
I know that, but the params `beta1`, `beta2` and `beta 3` only used in `main.py`, and you said 

> `parser.add_argument('--beta1', type=int, default=0, help='beta of low layer') parser.add_argument('--beta2', type=int, default=0, help='beta of middle layer') parser.add_argument('--beta3', type=int, default=0, help='beta of high layer')`
> After that finetuning the model by the command: `bash $ python train_badnet.py`.

**Comment by bboylyg** [comment]
Sorry, I take a mistake here. You should use the code of `main.py`. Thanks for your remind!
